MIP/mip-34/README.md

```diff
@@ -0,0 +1,270 @@
+# MIP-34: Fast-Finality Settlement
+
+- **Description**: Establish the scope and components that are part of the Fast-Finality Settlement mechanism.
+- **Authors**: [Franck Cassez](), [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+
+## Abstract
+
+Fast-Finality Settlement (FFS) is a mechanism that allows for fast _confirmation_ of transactions backed by crypto-economic security. This MIP outlines the high-level specifications and architecture of FFS.
+
+## Definitions
+
+- **L2-block** - See [Overview](#overview)
```

Comment:
imo an MG requires an MIP/MD since a new term needs to be introduced with reason. 

The definitions section raises the question, whether the MG should not be integrated into the MIP/MG. And the Glossary should reference the MIP/MG rather than the MG.